test/unit_tests/logging/test_logging_arkode_arkstep_lvl5_0.out

```diff
@@ -3,252 +3,207 @@ Using ERK method
            t                        u                        v                      u err                    v err            
 ------------------------------------------------------------------------------------------------------------------------------
  0.000000000000000e+00    1.224744871391589e+00    1.732050807568877e+00    0.000000000000000e+00    0.000000000000000e+00
-[INFO][rank 0][ARKodeEvolve][begin-step-attempt] step = 1, tn = 0, h = 0.0001029860256095084
+[INFO][rank 0][ARKodeEvolve][begin-step-attempt] step = 1, tn = 0, h = 0.000102986025609508
```

Comment:
It looks like the new approach consistently trims one character off of each number in comparison to what we'd previously printed.  Does anyone consider this to be an issue?

Reply:
I would make it one digit longer, but it's surprisingly tricky to add one to a macro and convert it to a string. One unintentional benefit is this may reduce the sensitivity of tests to failures due to minor floating point differences in out files.

Reply:
This is good with me. If we switch to `*_DECIMAL_DIG` rather than `*_DIG` in the future, we end up with more digits than previously printed.